libs/Fft.c

```diff
@@ -354,6 +357,9 @@ static void RenderCharFontSpec(void *p, XftCharFontSpec *sp)
 	if (batch->size == NR_CHARFONTSPEC || sp == NULL)
 	{
 		XftDrawCharFontSpec(batch->draw, batch->color, batch->specs, batch->size);
+		for (int i = 0; i < batch->size; i++)
```

Comment:
Some compilers will moan at this (C89 vs C99).   Since we're not consistent at all with C99, please pre-declare `int i` outside of this for loop.